server/views/pages/appointments/create-and-edit/bulk-appointments/upload-bulk-appointment.njk

```diff
@@ -18,17 +19,32 @@
                     id: "file",
                     name: "file",
                     label: {
-                        text: "Upload a CSV file to create appointments",
+                        text: "Upload your list of prison numbers",
                         classes: "govuk-label--l",
                         isPageHeading: true
                     },
+
+                    hint: {
+                        text: "Your list of prison numbers must be in a CSV file. You can include appointment start
+                        and end times using the 24-hour clock. For example 14:00 not 2:00pm"
+                    },
                     errorMessage: validationErrors | findError('file')
                 }) }}
-
+                {{ govukDetails({
+                  summaryText: "How to use a CSV file",
+                  text: howToUseCSVFile()
```

Comment:
While it does seem to work using text, the html property would be more appropriate due to the paragraph tags

Reply:
changed from text to html